packages/ui/src/AppHeader/SelectCluster/SelectCluster.tsx

```diff
@@ -0,0 +1,81 @@
+import cn from 'classnames'
+import React, { FC, useMemo } from 'react'
+import { CheckCircle } from 'react-feather'
+
+import { Loader } from '../../Loader'
+import { SelectField } from '../../Select'
+
+import styles from './SelectCluster.module.scss'
+
+const getSafeValue = (
+  clusterName: string | null,
+  clusterNames: string[] | null,
+): {
+  status: 'disabled' | 'connected' | 'disconnected'
+  value: string
+} => {
+  if (!clusterNames?.length) {
+    return { status: 'disabled', value: 'No Clusters' }
+  }
+
+  if (!clusterName) {
+    return {
+      status: 'disconnected',
+      value: clusterName || 'Select Cluster',
```

Comment:
Why do we have `clusterName` here?

Reply:
As is in MC

Reply:
It's definitely redundant. Let's remove it...

Reply:
done